### ISSUE TITLE
Search with multiple languages

### ISSUE THREAD

**Issue by kashmiry** [freearhey/knowledge-graph#2]
I used to be able to search with multiple languages like this: 
`$results = $graph->search($searchQuery, 'Person', 'en,ar', 3);`
But now this does not work, I think google made some changes to the query format from the console I can see its now like this: 
`GET https://kgsearch.googleapis.com/v1/entities:search?languages=en&languages=ar&query=messi&key={YOUR_API_KEY}
`

**Comment by kashmiry** [comment]
This is my current fix.
I use array to pass languages:
`$graph->search($searchQuery, 'Person', ['en,ar'], 3);
`
I added this function to Client.php, so I generate the query string with repeated 'languages' param, and pass it.

 ```
  public function request($params)
  {
    $query = array_merge($params, [
      'key' => $this->key
    ]);

    //Params to query string
    $walk = function( $item, $key, $parent_key = '' ) use ( &$output, &$walk ) {
        is_array( $item )
            ? array_walk( $item, $walk, $key )
            : $output[] = http_build_query( array( $parent_key ?: $key => $item ) );

    };
    $query = array_walk( $query, $walk );
    $query = implode( '&', $output );
    //Params to query string

    try {

      $response = $this->client->get(self::API_ENDPOINT, [
        'query' => $query
      ]);

    } catch (ClientException $exception) {

      return collect([]);

    }
    $results = json_decode($response->getBody(), true);

    $output = [];

    for($i = 0; $i < count($results['itemListElement']); $i++) {
      $output[] = new SearchResult($results['itemListElement'][$i]);
    }
      //$output[] = $response;
    return collect($output);
  }
```

**Comment by freearhey** [comment]
Thank you for reporting the problem. The bug is fixed in [v2.1.1](https://github.com/freearhey/knowledge-graph/releases/tag/v2.1.1)
